.changeset/cuddly-shrimps-fold.md

```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/clerk-react': patch
+'@clerk/types': patch
+---
+
+Add option to allow a `<SignIn />` component instance to opt into the sign-in-or-up flow.
```

Comment:
Guessing this should not be included in this PR?

Reply:
Yeah, if it's not part of your changes then it should not be included